index.js

```diff
@@ -149,10 +155,15 @@ SQLdown.prototype._get = function (key, options, cb) {
     if (!res.length) {
       return cb(new Error('NotFound'));
     }
+
     try {
       var value = JSON.parse(res[0].value);
       if (asBuffer) {
-        value = new Buffer(value);
+        if(value.type === 'Buffer'){
```

Comment:
We'll need to add this snippet from my PR to the _put so that it will round trip correctly encoded as utf8
[src](https://github.com/calvinmetcalf/SQLdown/blob/5aa3c43beb59725b8854ec76c4ff6d2ca5b78de1/index.js#L172-L174)
```js
  if (this._isBuffer(rawvalue)) {
    rawvalue = { type: 'Buffer', data: rawvalue.toString() }
  }
```